auto3dseg/metadata.json

```diff
@@ -1,6 +1,7 @@
 {
     "version": "0.0.1",
```

Comment:
Please also update the version to 0.0.2

Reply:
I have already added 0.0.2. Do you mean that I need to add 0.0.3?

Reply:
Hi @dongyang0122 , I mean the
```
"version": "0.0.1"
```
should be 
```
"version": "0.0.2"
```
since it is updated. Is that correct @yiheng-wang-nv ?

Reply:
Thanks. I have updated the `version`.